README.md

```diff
@@ -1,176 +1,98 @@
-
-```sh
-go get -d k8s.io/profiles-controller
-cd $GOPATH/src/k8s.io/profiles-controller
-godep restore
-```
-
-### When using go 1.11 modules
-
-When using go 1.11 modules (`GO111MODULE=on`), issue the following
-commands --- starting in whatever working directory you like.
-
-```sh
-git clone https://github.com/kubernetes/profiles-controller.git
-cd profiles-controller
-```
-
-Note, however, that if you intend to
-generate code then you will also need the
-code-generator repo to exist in an old-style location.  One easy way
-to do this is to use the command `go mod vendor` to create and
-populate the `vendor` directory.
-
-### A Note on kubernetes/kubernetes
-
-If you are developing Kubernetes according to
-https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md
-then you already have a copy of this demo in
-`kubernetes/staging/src/k8s.io/profiles-controller` and its dependencies
---- including the code generator --- are in usable locations
-(valid for all Go versions).
-
-## Purpose
-
-This is an example of how to build a kube-like controller with a single type.
-
-## Running
-
-**Prerequisite**: Since the profiles-controller uses `apps/v1` deployments, the Kubernetes cluster version should be greater than 1.9.
-
-```sh
-# assumes you have a working kubeconfig, not required if operating in-cluster
-go build -o profiles-controller .
-./profiles-controller -kubeconfig=$HOME/.kube/config
-
-# create a CustomResourceDefinition
-kubectl create -f artifacts/examples/crd.yaml
-
-# create a custom resource of type Foo
-kubectl create -f artifacts/examples/example-foo.yaml
-
-# check deployments created through the custom resource
-kubectl get deployments
-```
+## Controllers
 
-## Use Cases
+The [cmd](./cmd) package contains source files for a variety of profile controllers for Kubeflow.
 
-CustomResourceDefinitions can be used to implement custom resource types for your Kubernetes cluster.
-These act like most other Resources in Kubernetes, and may be `kubectl apply`'d, etc.
+For more information about profile controllers, see the documentation for the [client-go](https://github.com/kubernetes/client-go/) library, which contains a variety of mechanisms for use when
+developing custom profile controllers. The mechanisms are defined in the
+[tools/cache folder](https://github.com/kubernetes/client-go/tree/master/tools/cache) of the library.
 
-Some example use cases:
+### [authpolicy.go](./cmd/authpolicy.go)
 
-* Provisioning/Management of external datastores/databases (eg. CloudSQL/RDS instances)
-* Higher level abstractions around Kubernetes primitives (eg. a single Resource to define an etcd cluster, backed by a Service and a ReplicationController)
+Responsible for creating, removing and updating [Istio Authorization Policies](https://istio.io/latest/docs/reference/config/security/authorization-policy/#Source) using the [Istio client](https://github.com/istio/client-go) for a given `Profile`.
```

Comment:
Great! I would add as a small note, atm the only AuthPolicy is to block upload/download from protected-b notebooks

Reply:
Will do